docs/debugging-resources/org-debugging-resources.md

```diff
@@ -0,0 +1,21 @@
+# KubeVirt Org debugging resources
+
+## KubeVirt Prow
+
+[KubeVirt Prow] is the dedicated [Prow](https://docs.prow.k8s.io/docs/overview/) instance for the KubeVirt organization.
+
+## FlakeFinder
+
+[FlakeFinder] reports test failures from merged code to keep false positives at a minimum.
+
+## KubeVirt TestGrid
+
+In contrast to [FlakeFinder], [KubeVirt TestGrid] shows all test results from all prowjobs, including results from broken and unmerged PRs.
```

Comment:
You might not want to link to FlakeFinder in this section, to help avoid accidentally stealing a click from somebody trying to read about TestGrid.

Reply:
Makes absolute sense, thanks!